entrypoint.sh

```diff
@@ -4,27 +4,28 @@
 #  NAMESPACE=default POD_NAME=kubesh-3976960141-b9b9t ./this_script
 
 # Set VERBOSE=1 to get more output
+POD_NAME=${POD_NAME:-""}
 VERBOSE=${VERBOSE:-0}
-function verbose () {
-  [[ ${VERBOSE} -eq 1 ]] && return 0 || return 1
+verbose () {
+  [ "${VERBOSE}" -eq 1 ] && return 0 || return 1
 }
 
 echo 'This script polls the "EC2 Spot Instance Termination Notices" endpoint to gracefully stop and then reschedule all the pods running on this Kubernetes node, up to 2 minutes before the EC2 Spot Instance backing this node is terminated.'
 echo 'See https://aws.amazon.com/blogs/aws/new-ec2-spot-instance-termination-notices/ for more information.'
 
-if [ "${NAMESPACE}" == "" ]; then
-  echo '[ERROR] Environment variable `NAMESPACE` has no value set. You must set it via PodSpec like described in http://stackoverflow.com/a/34418819' 1>&2
+if [ "${NAMESPACE}" = "" ]; then
+  echo "[ERROR] Environment variable $(NAMESPACE) has no value set. You must set it via PodSpec like described in http://stackoverflow.com/a/34418819" 1>&2
```

Comment:
Isn't `$(NAMESPACE)` incorrect?

Reply:
@Multiply Rechecked it, backticks offer the same behavior the same as `$()` so I guess this is actually ok. Shellcheck offered this, wdyt?

Reply:
Because you're now using `"` instead of the previously used `'`.

```
$ echo '[ERROR] Environment variable `NAMESPACE` is wrong'
[ERROR] Environment variable `NAMESPACE` is wrong

 $ echo "[ERROR] Environment variable `NAMESPACE` is wrong"
-bash: NAMESPACE: command not found
[ERROR] Environment variable  is wrong
```

I didn't use the same string as mentioned above, as this was just to prove my point.

Reply:
@Multiply Got it, so it's not the backticks, it's the just using `'`.
Will update my changes